roles/cloud-digitalocean/tasks/main.yml

```diff
@@ -60,8 +60,16 @@
         ipv6: yes
       register: do
 
+    - name: "Assigning floating IP..."
+      digital_ocean_floating_ip:
+        oauth_token: "{{ algo_do_token }}"
+        ip: "{{ cloud_providers.digitalocean.floating_ip }}"
```

Comment:
I don't think we are going to use any existing floating IPs, but allocate a new one instead. We souldn't depend on any existing user's infrastructure

Reply:
so my original motivation here was to simplify maintenance by making the algo server have the same public ip address every time, and so i did it this way because creating a new floating ip address every time wouldn't accomplish that (honestly, i have no idea what it actually would accomplish).

that said, i've improved my provisioning process enough that it's not really a meaningful issue for me anymore, so i'm just going to close this instead.